qiskit/qobj/converters/pulse_instruction.py

```diff
@@ -341,6 +341,24 @@ def convert_shift_phase(self, shift, instruction):
         }
         return self._qobj_model(**command_dict)
 
+    @bind_instruction(instructions.Delay)
+    def convert_shift_phase(self, shift, instruction):
+        """Return converted `ShiftPhase`.
+
+        Args:
+            shift(int): Offset time.
+            instruction (ShiftPhase): Shift phase instruction.
```

Comment:
The name for this method needs to be updated.

Reply:
😨 😱  So bad, thank you